src/styles/index.scss

```diff
@@ -1,3 +1,77 @@
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vmin;
   margin: 0;
 }
+
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border-radius: 50%;
+  border: 1vmin dotted black;
+}
+
+.stopwatch__center {
+  position: absolute;
+
+  border-radius: 50%;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 5vmin;
+  height: 5vmin;
+  background-color: #f6a603;
+}
+
+.stopwatch__minutes-hand {
+  position: absolute;
+  top: 24%;
+  left: 50%;
+  width: 3vmin;
+  height: 20vmin;
+  background-color: #0700ff;
+  transform-origin: 50% 100%;
+
+  animation: minutes 3600s steps(60) infinite;
```

Comment:
The `steps(60)` function in the `minutes` animation may cause performance issues on some devices due to the long duration and the number of steps. Consider using a different timing function if the animation appears to be janky or is not performing well.